docs/api/_index.md

```diff
@@ -123,6 +124,14 @@ GET /config?mode=defaults
 
 Displays the configuration using only the default values.
 
+### Runtime Configuration
+
+```
+GET /runtime_config
+```
+
+Displays the runtime configuration currently applied to Cortex (in YAML format), including default values. Please be aware that the configuration will be available only if Cortex is configured with a valid `-runtime-config.file`.
```

Comment:
`-runtime-config.file` option must be set for this endpoint to be available, but actual file doesn’t need to exist or be valid. But that’s just nitpicking.